CeeFM/Decideify
# Content Hub Suggestion Forms Return Unique Suggestions (No Duplicates) And Require Refresh When They Run Out of Unique Suggestions

**Description**
Just like the title says! We had problems with repeat suggestions and never-ending suggestion cycles. Now we don't! Or, should I say, we shouldn't!

**Type of Change**


**Testing Instructions**

- Clone this project’s repository to your machine
  -If you’ve already done this and are currently working on a different branch of the project, be sure to `git add` and `git commit` whatever you’re working on on your branch, before you attempt to switch branches
  -Then `git checkout main` to switch to the project’s main branch)
- Navigate to the root directory for this repo (yourParentFolderName/Decideify)
- `git fetch -a` to retrieve the current remote-tracked branches for the project, you should see a branch named `cm-suggestions` as one of the available branches
- `git checkout cm-suggestions` to switch to my branch
- If you don't see the most recent version of my files, `git pull origin cm-suggestions`
- `Start Decideify.sln` to open the project in Visual Studio
- If you haven’t created the Decideify database, please do so in Visual Studio and use the following SQL scripts to populate it accordingly:
  - Start a new query in your database and copy in this SQL script before running to fill out tables and columns: [First SQL Script for Creating DB](https://github.com/CeeFM/Decideify/blob/main/SQL/Decideify_Create_DB.sql)
  - ctrl + a and delete what is in that query and replace it with this SQL script before running to seed the database: [Second SQL Script for Seeding DB](https://github.com/CeeFM/Decideify/blob/main/SQL/Decideify_Seed_Users_DB.sql)

- Click the execute with debugger solid green button to launch the app
- On any content hub (/books, /movies, /tv, or /music):
  - Submitting the content suggestion form will follow the basic flow as it did previously
  - Once suggestions are rendering to the browser, and you can cycle through them, you should not see any duplicate suggestions while clicking through the options.
  - Additionally, when all suggestions from the API call's data have been cycled through, a message will show up where the suggestions have been rendering that informs the user there are no suggestions left for this search, and they need to start again.

Checklist
